debian/Dockerfile

```diff
@@ -34,16 +34,18 @@ RUN set -ex && \
       apt-key adv --no-tty --batch --keyserver hkp://keyserver.pgp.com:80 --recv-keys "$key" || \
       apt-key adv --no-tty --batch --keyserver hkp://ipv4.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
     done && \
-    echo "deb https://apt.dockerproject.org/repo debian-jessie main" > /etc/apt/sources.list.d/docker.list && \
+    curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - && \
+    echo "deb https://download.docker.com/linux/debian buster stable" > /etc/apt/sources.list.d/docker.list && \
     apt-get update && \
-    apt-get install -y docker-engine && \
+    apt-get install -y docker-ce docker-ce-cli && \
     pip3 install --upgrade \
         awscli \
         virtualenv \
         && \
     mkdir -p ${JENKINS_HOME} && \
     useradd --home ${JENKINS_HOME} --system jenkins && \
     chown -R jenkins ${JENKINS_HOME} && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
```

Comment:
`python` isn't a command anymore (only `python3`), so adding that symlink back so it doesn't bite anyone.